### PR TITLE
removed all static variables user_id

### PR DESCRIPTION
**Pull request by matviichuk92**

### CODE REVIEW

**Review by AnaLanda — COMMENTED** [review]
Looks good! :smiley:

File: src/main/java/online/shop/controller/AddProductToShoppingCartController.java [hunk 1]
```diff
@@ -22,7 +21,8 @@
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        ShoppingCart shoppingCart = shoppingCartService.getByUserId(USER_ID);
+        Long userId = (Long) req.getSession().getAttribute("userId");
```
Comment (mikhail771) [inline src/main/java/online/shop/controller/AddProductToShoppingCartController.java]: Good job. "user_id" likes good to be a constant. Check, please, in all changed controllers

**Comment by mikhail771** [comment]
mentors said to post links to "Files changed" in google docs

File: src/main/java/online/shop/controller/AddProductToShoppingCartController.java (hunk 1 above)
Comment (KseniiaMakarova) [reply src/main/java/online/shop/controller/AddProductToShoppingCartController.java]: it really looks slightly better when moved into a constant

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**